subworkflows/local/vcf_annotate_bcftools/main.nf

```diff
@@ -8,15 +8,12 @@ include { TABIX_TABIX       } from '../../../modules/nf-core/tabix/tabix/main'
 
 workflow VCF_ANNOTATE_BCFTOOLS {
     take:
-    vcf               // channel: [ val(meta), vcf ]
-    annotations       //
-    annotations_index //
-    header_lines      //
+    ch_vcf               // channel: [ val(meta), vcf ]
 
     main:
     ch_versions = Channel.empty()
 
-    BCFTOOLS_ANNOTATE(vcf, annotations, annotations_index, header_lines)
+    BCFTOOLS_ANNOTATE(ch_vcf.map { meta, vcf -> [ meta, vcf, [], [], [], []]})
```

Comment:
what happened to `annotations, annotations_index, header_lines`?

Reply:
It wasn't used in the pipeline, I might take a look at properly adding it in another PR

Reply:
ok